tests/test_serializers.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import unicode_literals
+
+from decimal import Decimal
+
+from django.test import TestCase
+from django.utils import timezone
+from django.conf import settings
+
+from mock import patch, PropertyMock
+from djstripe.serializers import (
```

Comment:
This would be cleaner on one line. There are similar instances in your other files.